assets/stylesheets/shared/_colors.scss

```diff
@@ -1,74 +1,77 @@
+/**
+ * Colors
+ *
+ * Color variables based on the design handbook
+ * See wordpress.com/design-handbook/colors/ for more info.
+ */
+
 // Primary Accent (Blues)
 $blue-wordpress:         #0087be;
 $blue-light:             #78dcfa;
 $blue-medium:            #00aadc;
 $blue-dark:              #005082;
 
-// Jetpack Green
-$green-jetpack:          #8cc258;
-
 // Grays
 $gray:                   #87a6bc;
 $gray-light:             lighten( $gray, 33% ); //#f3f6f8
+$gray-lighten-30:        lighten( $gray, 30% ); //#e9eff3
+$gray-lighten-20:        lighten( $gray, 20% ); //#c8d7e1
+$gray-lighten-10:        lighten( $gray, 10% ); //#a8bece
+$gray-darken-10:         darken( $gray, 10% ); //#668eaa
+$gray-darken-20:         darken( $gray, 20% ); //#4f748e
+$gray-darken-30:         darken( $gray, 30% ); //#3d596d
```

Comment:
We didn't want the % values to be too limiting, so we didn't make them variables. It was more intended for $gray, because we needed to enabled designers to pick the shade of gray they needed. We probably shouldn't have so many different %s of WordPress Blue.

That being said, I'm not totally against using variables for them if it's necessary. It doesn't prohibit us from using a custom % when we want to. I agree, maybe a separate PR would be better.